zap/src/main/java/org/parosproxy/paros/extension/ExtensionHookMenu.java

```diff
@@ -245,6 +245,16 @@ public void addFileMenuItem(JMenuItem menuItem) {
         addFileMenuItemImpl(menuItem);
     }
 
+    /**
+     * Add a sub-menu to the File menu
+     *
+     * @param menu the sub-menu to add
+     * @since TODO Add version
+     */
+    public void addFileMenuItem(JMenu menu) {
```

Comment:
`MenuItem` → `Menu`?

Reply:
I was worried about naming confusion, it isn't adding the File Menu. Maybe `addFileSubMenu`?